newswires/client/src/useSearch.tsx

```diff
@@ -291,7 +297,12 @@ export function SearchContextProvider({ children }: PropsWithChildren) {
 		if (state.status === 'success') {
 			pollingInterval = setInterval(() => {
 				// Poll for updated results
-				fetchResults(currentConfig.query)
+				fetchResults(
+					currentConfig.query,
+					Math.max(
+						...state.queryData.results.map((wire) => wire.id),
```

Comment:
let's not rely on the id being a monotonically increasing value - instead let's find the max by the attribute we actually want to sort on (probably one of the dates) - or just choose whichever content is in the 0th position

Reply:
I'm happy with that in principle, but I feel like it makes sense given that this is the way the back-end API [currently works](https://github.com/guardian/newswires/blob/195ea1f3edd9f9f4e85398ec8938f2c7d1715b4d/newswires/app/db/FingerpostWireEntry.scala#L149-L151)?

Reply:
gah, great point - let's address this later